connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSourceTaskTest.java

```diff
@@ -315,33 +305,28 @@ public void testPause() throws Exception {
 
         taskFuture.get();
 
-        PowerMock.verifyAll();
+        verifyCleanStartup();
+        verifyTaskGetTopic(10);
+        verifyOffsetFlush(true);
+        verifyTopicCreation(TOPIC);
+        verify(statusListener).onPause(taskId);
+        verify(statusListener).onShutdown(taskId);
+        verify(sourceTask).stop();
+        verify(offsetWriter).offset(PARTITION, OFFSET);
+        verifyClose();
     }
 
     @Test
     public void testPollsInBackground() throws Exception {
         createWorkerTask();
 
-        expectCleanStartup();
-
         final CountDownLatch pollLatch = expectPolls(10);
         // In this test, we don't flush, so nothing goes any further than the offset writer
 
         expectTopicCreation(TOPIC);
+        expectOffsetFlush(true, true);
```

Comment:
Nit: this can be simplified, right?
```suggestion
        expectOffsetFlush(true);
```